Calibration/HcalCalibAlgos/plugins/RecAnalyzerMinbias.cc

```diff
@@ -280,9 +285,12 @@ void RecAnalyzerMinbias::fillDescriptions(edm::ConfigurationDescriptions& descri
      histo_.push_back(hh);
    };
 
-   h_AmplitudeHBtest = fs_->make<TH1D>("h_AmplitudeHBtest", "", 320, 0., 100.);
-   h_AmplitudeHEtest = fs_->make<TH1D>("h_AmplitudeHEtest", "", 320, 0., 100.);
-   h_AmplitudeHFtest = fs_->make<TH1D>("h_AmplitudeHFtest", "", 320, 0., 100.);
+   h_AmplitudeHBtest_ = fs_->make<TH1D>("h_AmplitudeHBtest","", 5000, 0., 5000.);
+   h_AmplitudeHEtest_ = fs_->make<TH1D>("h_AmplitudeHEtest","",300000,0.,300000.);
```

Comment:
@bsunanda why such a big increase in the number of bins? Are they really needed? And in which workflow is this code supposed to run?